Description: Move the declaration of "_synctex_open", as it is needed by an earlier function.
Author: Pino Toscano <pino@kde.org>
Index: synctex/synctex_parser.c
===================================================================
--- synctex.orig/synctex_parser.c	2010-05-23 18:52:35.000000000 +0200
+++ synctex/synctex_parser.c	2010-05-23 18:54:04.000000000 +0200
@@ -2481,6 +2481,8 @@
 	goto next_sheet;
 }
 
+int _synctex_open(const char * output, const char * build_directory, char ** synctex_name_ref, gzFile * file_ref, synctex_bool_t add_quotes, synctex_io_mode_t * io_modeRef);
+
 /*  Where the synctex scanner is created. */
 synctex_scanner_t synctex_scanner_new_with_output_file(const char * output, const char * build_directory, int parse) {
 	gzFile file = NULL;
@@ -2650,8 +2652,6 @@
 #	undef io_mode
 }
 
-int _synctex_open(const char * output, const char * build_directory, char ** synctex_name_ref, gzFile * file_ref, synctex_bool_t add_quotes, synctex_io_mode_t * io_modeRef);
-
 /*	Opens the ouput file, taking into account the eventual build_directory.
  *	0 on success, non 0 on error. */
 int _synctex_open(const char * output, const char * build_directory, char ** synctex_name_ref, gzFile * file_ref, synctex_bool_t add_quotes, synctex_io_mode_t * io_modeRef) {
